website/docs/Installation/windows-install.md

```diff
@@ -3,69 +3,44 @@ sidebar_position: 4
 ---
 
 # Windows
-## 1. Installing Podman Desktop on Windows
 
-In order to install the latest Podman Desktop application for Windows, visit the [Downloads](/downloads/windows) section of this website to download the .exe file.
+This page contains information regarding installation of Podman Desktop on Windows.
 
-Simply download the file from the [Downloads](/downloads/windows) section and open it in your Desktop to install Podman Desktop.
+:::infoPrerequisite
+Podman Desktop on Windows requires the following:
+1. [Podman Engine](https://docs.podman.io/en/latest/index.html)
+2. Hyper-V should be enabled
```

Comment:
link to what is Hyper-V, how to enable it or install it

Reply:
added